Cars/Cars/CarBodyType.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Cars
+{
+    public enum CarBodyType
+    {
+            Sedan,
+            Hetchback,
```

Comment:
По-английски правильно
```suggestion
            Hatchback,
```